src/Persistence.Tests/PaYaml/Serialization/PaYamlSerializerTests.cs

```diff
@@ -214,5 +214,15 @@ public void IsSequenceCheckShouldWorkAsExpected(string path, bool expected)
         isSequence.Should().Be(expected);
     }
 
+    [TestMethod]
+    [DataRow("")]
+    [DataRow("   ")]
+    [DataRow("not yaml")]
+    public void IsSequenceCheckShouldReturnFalseWhenYamlInvalid(string yaml)
```

Comment:
also add tests for inline syntaxes:
- `[]`
- `['str`,333]`
- `{ prop1: str2 }`